master_visu_scripts/vtk_plot.py

```diff
@@ -11,48 +11,37 @@
 from tqdm import tqdm
 from cmcrameri import cm
 
-def plot(file,field='density',bounds=None,contours=False,
+def plot(file,field,bounds,ax=None,contours=False,
          cfields=['crust_upper','crust_lower','mantle_lithosphere'],
-         null_field='asthenosphere',off_screen=True,output='img.png',
-         camera=None,plot_scalar_bar=True,**kwargs):
+         null_field='asthenosphere',plot_scalar_bar=True,**kwargs):

```

Comment:
`camera` and `off_screen` should also be removed from the plot call in `stress_plots.py`

Reply:
This has been addressed by d09ccf2 and 0b41c06.